doc/api/index.md

```diff
@@ -169,6 +170,54 @@ player.loadVideo({
 });
 ```
 
+<a name="meth-reload"></a>
+### reload #####################################################################
+
+_arguments_:
+  - _options_ (``Object``)
+
+Re-load the last loaded content, with its original loadVideo options, as fast as
+possible, using the last loaded manifest. The aim is to reduce loading time by
+removing the fetching and parsing steps.
```

Comment:
Are we sure that there is nothing more we could do in the future?
I would remove all notions about the fact that this methods just stores and re-use the last Manifest because it blocks what its implementation can be.

We coud even imagine cases in the future where we would still need to fetch the Manifest.

Reply:
Why not, but the API description would just be "reload the content" without any indication on what are the benefits ?

Reply:
We could just be saying in some way that this can be faster than `loadVideo` (or at least, never slower).